GliaWidgets/Sources/RemoteConfiguration/RemoteConfiguration.swift

```diff
@@ -36,14 +36,20 @@ extension RemoteConfiguration {
         let header: Header?
         let callOperator, topText: Text?
         let connect: EngagementStates?
+        let visitorVideo: VisitorVideo?
 
         enum CodingKeys: String, CodingKey {
             case background, bottomText, buttonBar, duration, header, connect
             case callOperator = "operator"
             case topText
+            case visitorVideo
         }
     }
 
+    struct VisitorVideo: Codable {
+        let flipCameraButton: BarButtonStates?
```

Comment:
@AndriiHorishniiMOC has opened [PR](https://github.com/salemove/mobile-sdk-unified/pull/36/files) to remove redundant button styles to have only one state. 
@igorkravchenko are you going to align it on WidgetsSDK side in next PR?

Reply:
Sure, we had discussion with Andrii about it. Thank you for your help with this PR 🤝

Reply:
Updated 05f24ae04c2e1d1138859cb87a1a4ee052acaf3f.